src/dsp_tools/commands/xmlupload/models/serialise/serialise_value.py

```diff
@@ -48,9 +59,39 @@ def serialise(self) -> dict[str, Any]:
         return serialised
 
 
+class SerialiseDateValue(SerialiseValue):
```

Comment:
The others don't end in ...Value

```suggestion
class SerialiseDate(SerialiseValue):
```